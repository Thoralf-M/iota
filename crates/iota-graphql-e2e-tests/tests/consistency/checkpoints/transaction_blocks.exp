processed 16 tasks

init:
A: object(0,0), B: object(0,1)

task 1, lines 14-27:
//# publish
created: object(1,0)
mutated: object(0,2)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 5168000,  storage_rebate: 0, non_refundable_storage_fee: 0

task 2, line 29:
//# run Test::M1::create --args 0 @A --sender A
created: object(2,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 0, non_refundable_storage_fee: 0

task 3, line 31:
//# run Test::M1::create --args 0 @A --sender A
created: object(3,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 4, line 33:
//# run Test::M1::create --args 0 @A --sender A
created: object(4,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 5, line 35:
//# run Test::M1::create --args 0 @A --sender A
created: object(5,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 6, line 37:
//# create-checkpoint
Checkpoint created: 1

task 7, line 39:
//# run Test::M1::create --args 0 @A --sender A
created: object(7,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 8, line 41:
//# run Test::M1::create --args 0 @A --sender A
created: object(8,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 9, line 43:
//# run Test::M1::create --args 0 @A --sender A
created: object(9,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 10, line 45:
//# create-checkpoint
Checkpoint created: 2

task 11, line 47:
//# run Test::M1::create --args 0 @A --sender A
created: object(11,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 12, line 49:
//# run Test::M1::create --args 0 @A --sender A
created: object(12,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 13, line 51:
//# create-checkpoint
Checkpoint created: 3

task 14, lines 53-77:
//# run-graphql
Response: {
  "data": {
    "checkpoints": {
      "nodes": [
        {
          "sequenceNumber": 0,
          "transactionBlocks": {
            "edges": []
          }
        },
        {
          "sequenceNumber": 1,
          "transactionBlocks": {
            "edges": [
              {
                "cursor": "eyJjIjozLCJ0IjoyLCJpIjpmYWxzZX0",
                "node": {
                  "digest": "B1nkiuMBtC8ZXZLmL6rYBE81wBnxSSfnk2nhCgo7FCPB",
                  "sender": {
                    "objects": {
                      "edges": [
                        {
                          "cursor": "INyOf5AGhch4zxokSXXqjyUjFp+WvdTBbe1pjdI3OJwRAwAAAAAAAAA="
                        }
                      ]
                    }
                  }
                }
              },
              {
                "cursor": "eyJjIjozLCJ0IjozLCJpIjpmYWxzZX0",
                "node": {
                  "digest": "7XN9xEUZbYPMA8E3gNrNLqHZMFf8FtEZTUjz4cAVBQCc",
                  "sender": {
                    "objects": {
                      "edges": [
                        {
                          "cursor": "INyOf5AGhch4zxokSXXqjyUjFp+WvdTBbe1pjdI3OJwRAwAAAAAAAAA="
                        }
                      ]
                    }
                  }
                }
              },
              {
                "cursor": "eyJjIjozLCJ0Ijo0LCJpIjpmYWxzZX0",
                "node": {
                  "digest": "6YdycjvxF5xZMwpMDsgB91EcnshoaKCQsv7b6njXedqS",
                  "sender": {
                    "objects": {
                      "edges": [
                        {
                          "cursor": "INyOf5AGhch4zxokSXXqjyUjFp+WvdTBbe1pjdI3OJwRAwAAAAAAAAA="
                        }
                      ]
                    }
                  }
                }
              },
              {
                "cursor": "eyJjIjozLCJ0Ijo1LCJpIjpmYWxzZX0",
                "node": {
                  "digest": "3cEHRKQYeXwiVVGxhCSnEdQAt6eGd3xgHPq6zMHY5YJ4",
                  "sender": {
                    "objects": {
                      "edges": [
                        {
                          "cursor": "INyOf5AGhch4zxokSXXqjyUjFp+WvdTBbe1pjdI3OJwRAwAAAAAAAAA="
                        }
                      ]
                    }
                  }
                }
              }
            ]
          }
        },
        {
          "sequenceNumber": 2,
          "transactionBlocks": {
            "edges": [
              {
                "cursor": "eyJjIjozLCJ0Ijo2LCJpIjpmYWxzZX0",
                "node": {
                  "digest": "A5LGGScz3Bihtb6VKmRVMm1n3oVW18qPUu7v4w7H4fBp",
                  "sender": {
                    "objects": {
                      "edges": [
                        {
                          "cursor": "INyOf5AGhch4zxokSXXqjyUjFp+WvdTBbe1pjdI3OJwRAwAAAAAAAAA="
                        }
                      ]
                    }
                  }
                }
              },
              {
                "cursor": "eyJjIjozLCJ0Ijo3LCJpIjpmYWxzZX0",
                "node": {
                  "digest": "H9Aze5HhVxnWNbmCSxQMznv5PHaqZFqKvQvYtRChF53N",
                  "sender": {
                    "objects": {
                      "edges": [
                        {
                          "cursor": "INyOf5AGhch4zxokSXXqjyUjFp+WvdTBbe1pjdI3OJwRAwAAAAAAAAA="
                        }
                      ]
                    }
                  }
                }
              },
              {
                "cursor": "eyJjIjozLCJ0Ijo4LCJpIjpmYWxzZX0",
                "node": {
                  "digest": "ARoDw9b8ZrnDtL3iWQivXXNATncRMDqG5w9HzpgSvEfG",
                  "sender": {
                    "objects": {
                      "edges": [
                        {
                          "cursor": "INyOf5AGhch4zxokSXXqjyUjFp+WvdTBbe1pjdI3OJwRAwAAAAAAAAA="
                        }
                      ]
                    }
                  }
                }
              }
            ]
          }
        },
        {
          "sequenceNumber": 3,
          "transactionBlocks": {
            "edges": [
              {
                "cursor": "eyJjIjozLCJ0Ijo5LCJpIjpmYWxzZX0",
                "node": {
                  "digest": "FzrKhqbJPfqNMuoPzw1UThN8ttYNCxDNpf6XSo3BbScq",
                  "sender": {
                    "objects": {
                      "edges": [
                        {
                          "cursor": "INyOf5AGhch4zxokSXXqjyUjFp+WvdTBbe1pjdI3OJwRAwAAAAAAAAA="
                        }
                      ]
                    }
                  }
                }
              },
              {
                "cursor": "eyJjIjozLCJ0IjoxMCwiaSI6ZmFsc2V9",
                "node": {
                  "digest": "2CKkQpstA1cMNEwah1G8oyGRsHFTuoJ5DdzK7e2cukWG",
                  "sender": {
                    "objects": {
                      "edges": [
                        {
                          "cursor": "INyOf5AGhch4zxokSXXqjyUjFp+WvdTBbe1pjdI3OJwRAwAAAAAAAAA="
                        }
                      ]
                    }
                  }
                }
              }
            ]
          }
        }
      ]
    }
  }
}

task 15, lines 79-96:
//# run-graphql
Response: {
  "data": {
    "checkpoints": {
      "nodes": [
        {
          "sequenceNumber": 0,
          "epoch": {
            "epochId": 0,
            "checkpoints": {
              "nodes": [
                {
                  "sequenceNumber": 0
                },
                {
                  "sequenceNumber": 1
                },
                {
                  "sequenceNumber": 2
                },
                {
                  "sequenceNumber": 3
                }
              ]
            }
          }
        },
        {
          "sequenceNumber": 1,
          "epoch": {
            "epochId": 0,
            "checkpoints": {
              "nodes": [
                {
                  "sequenceNumber": 0
                },
                {
                  "sequenceNumber": 1
                },
                {
                  "sequenceNumber": 2
                },
                {
                  "sequenceNumber": 3
                }
              ]
            }
          }
        },
        {
          "sequenceNumber": 2,
          "epoch": {
            "epochId": 0,
            "checkpoints": {
              "nodes": [
                {
                  "sequenceNumber": 0
                },
                {
                  "sequenceNumber": 1
                },
                {
                  "sequenceNumber": 2
                },
                {
                  "sequenceNumber": 3
                }
              ]
            }
          }
        },
        {
          "sequenceNumber": 3,
          "epoch": {
            "epochId": 0,
            "checkpoints": {
              "nodes": [
                {
                  "sequenceNumber": 0
                },
                {
                  "sequenceNumber": 1
                },
                {
                  "sequenceNumber": 2
                },
                {
                  "sequenceNumber": 3
                }
              ]
            }
          }
        }
      ]
    }
  }
}
