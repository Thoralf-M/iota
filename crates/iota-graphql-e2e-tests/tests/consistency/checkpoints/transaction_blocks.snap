---
source: external-crates/move/crates/move-transactional-test-runner/src/framework.rs
---
processed 16 tasks

init:
A: object(0,0), B: object(0,1)

task 1, lines 14-27:
//# publish
created: object(1,0)
mutated: object(0,2)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 5168000,  storage_rebate: 0, non_refundable_storage_fee: 0

task 2, line 29:
//# run Test::M1::create --args 0 @A --sender A
created: object(2,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 0, non_refundable_storage_fee: 0

task 3, line 31:
//# run Test::M1::create --args 0 @A --sender A
created: object(3,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 4, line 33:
//# run Test::M1::create --args 0 @A --sender A
created: object(4,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 5, line 35:
//# run Test::M1::create --args 0 @A --sender A
created: object(5,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 6, line 37:
//# create-checkpoint
Checkpoint created: 1

task 7, line 39:
//# run Test::M1::create --args 0 @A --sender A
created: object(7,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 8, line 41:
//# run Test::M1::create --args 0 @A --sender A
created: object(8,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 9, line 43:
//# run Test::M1::create --args 0 @A --sender A
created: object(9,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 10, line 45:
//# create-checkpoint
Checkpoint created: 2

task 11, line 47:
//# run Test::M1::create --args 0 @A --sender A
created: object(11,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 12, line 49:
//# run Test::M1::create --args 0 @A --sender A
created: object(12,0)
mutated: object(0,0)
gas summary: computation_cost: 1000000, computation_cost_burned: 1000000, storage_cost: 2287600,  storage_rebate: 980400, non_refundable_storage_fee: 0

task 13, line 51:
//# create-checkpoint
Checkpoint created: 3

task 14, lines 53-77:
//# run-graphql
Response: {
  "data": {
    "checkpoints": {
      "nodes": [
        {
          "sequenceNumber": 0,
          "transactionBlocks": {
            "edges": []
          }
        },
        {
          "sequenceNumber": 1,
          "transactionBlocks": {
            "edges": [
              {
                "cursor": "eyJjIjozLCJ0IjoyLCJpIjpmYWxzZX0",
                "node": {
                  "digest": "AqKeLka2psDxT5mKGaW3BtMtsZSaBAvePSgneTBhMaM3",
                  "sender": {
                    "objects": {
                      "edges": [
                        {
                          "cursor": "INZmzdmWPU/lOSO6+ZIJf5dFU6m78tdMLLQs96g3v73qAwAAAAAAAAA="
                        }
                      ]
                    }
                  }
                }
              },
              {
                "cursor": "eyJjIjozLCJ0IjozLCJpIjpmYWxzZX0",
                "node": {
                  "digest": "2Xo3P52jeiqttVZUE5MyTdwbSjjcgGCuFgYtF74P5Cru",
                  "sender": {
                    "objects": {
                      "edges": [
                        {
                          "cursor": "INZmzdmWPU/lOSO6+ZIJf5dFU6m78tdMLLQs96g3v73qAwAAAAAAAAA="
                        }
                      ]
                    }
                  }
                }
              },
              {
                "cursor": "eyJjIjozLCJ0Ijo0LCJpIjpmYWxzZX0",
                "node": {
                  "digest": "5ZFxt4Vo5w6478FCiR2yGG16JyZC6HeWBZFL28JKU2of",
                  "sender": {
                    "objects": {
                      "edges": [
                        {
                          "cursor": "INZmzdmWPU/lOSO6+ZIJf5dFU6m78tdMLLQs96g3v73qAwAAAAAAAAA="
                        }
                      ]
                    }
                  }
                }
              },
              {
                "cursor": "eyJjIjozLCJ0Ijo1LCJpIjpmYWxzZX0",
                "node": {
                  "digest": "5i9zqxCjr7rkhA2uYaHrrJZn8uFE8jvjwUQf3dA8kA6b",
                  "sender": {
                    "objects": {
                      "edges": [
                        {
                          "cursor": "INZmzdmWPU/lOSO6+ZIJf5dFU6m78tdMLLQs96g3v73qAwAAAAAAAAA="
                        }
                      ]
                    }
                  }
                }
              }
            ]
          }
        },
        {
          "sequenceNumber": 2,
          "transactionBlocks": {
            "edges": [
              {
                "cursor": "eyJjIjozLCJ0Ijo2LCJpIjpmYWxzZX0",
                "node": {
                  "digest": "AoorSccqwZCGgiPQgNzSRGqNBhvR18iwF1UboCzAUCoj",
                  "sender": {
                    "objects": {
                      "edges": [
                        {
                          "cursor": "INZmzdmWPU/lOSO6+ZIJf5dFU6m78tdMLLQs96g3v73qAwAAAAAAAAA="
                        }
                      ]
                    }
                  }
                }
              },
              {
                "cursor": "eyJjIjozLCJ0Ijo3LCJpIjpmYWxzZX0",
                "node": {
                  "digest": "Ak8JCxSwJrVGeGyGShsRbF2koRxbYB7u3yhiKkzEhg5x",
                  "sender": {
                    "objects": {
                      "edges": [
                        {
                          "cursor": "INZmzdmWPU/lOSO6+ZIJf5dFU6m78tdMLLQs96g3v73qAwAAAAAAAAA="
                        }
                      ]
                    }
                  }
                }
              },
              {
                "cursor": "eyJjIjozLCJ0Ijo4LCJpIjpmYWxzZX0",
                "node": {
                  "digest": "rTtcpSJm2DJ1joEVd5JhoPBgmntTHcpNz1mDMLw99E8",
                  "sender": {
                    "objects": {
                      "edges": [
                        {
                          "cursor": "INZmzdmWPU/lOSO6+ZIJf5dFU6m78tdMLLQs96g3v73qAwAAAAAAAAA="
                        }
                      ]
                    }
                  }
                }
              }
            ]
          }
        },
        {
          "sequenceNumber": 3,
          "transactionBlocks": {
            "edges": [
              {
                "cursor": "eyJjIjozLCJ0Ijo5LCJpIjpmYWxzZX0",
                "node": {
                  "digest": "BRiR7LcVSsN3eGTPQx3FccBk2Gb6vuPFd4EKKYG67pG6",
                  "sender": {
                    "objects": {
                      "edges": [
                        {
                          "cursor": "INZmzdmWPU/lOSO6+ZIJf5dFU6m78tdMLLQs96g3v73qAwAAAAAAAAA="
                        }
                      ]
                    }
                  }
                }
              },
              {
                "cursor": "eyJjIjozLCJ0IjoxMCwiaSI6ZmFsc2V9",
                "node": {
                  "digest": "C9RmvpeqmMNYyjMM3jAx2bHWFxgo5KpmHucWdkakWtJd",
                  "sender": {
                    "objects": {
                      "edges": [
                        {
                          "cursor": "INZmzdmWPU/lOSO6+ZIJf5dFU6m78tdMLLQs96g3v73qAwAAAAAAAAA="
                        }
                      ]
                    }
                  }
                }
              }
            ]
          }
        }
      ]
    }
  }
}

task 15, lines 79-96:
//# run-graphql
Response: {
  "data": {
    "checkpoints": {
      "nodes": [
        {
          "sequenceNumber": 0,
          "epoch": {
            "epochId": 0,
            "checkpoints": {
              "nodes": [
                {
                  "sequenceNumber": 0
                },
                {
                  "sequenceNumber": 1
                },
                {
                  "sequenceNumber": 2
                },
                {
                  "sequenceNumber": 3
                }
              ]
            }
          }
        },
        {
          "sequenceNumber": 1,
          "epoch": {
            "epochId": 0,
            "checkpoints": {
              "nodes": [
                {
                  "sequenceNumber": 0
                },
                {
                  "sequenceNumber": 1
                },
                {
                  "sequenceNumber": 2
                },
                {
                  "sequenceNumber": 3
                }
              ]
            }
          }
        },
        {
          "sequenceNumber": 2,
          "epoch": {
            "epochId": 0,
            "checkpoints": {
              "nodes": [
                {
                  "sequenceNumber": 0
                },
                {
                  "sequenceNumber": 1
                },
                {
                  "sequenceNumber": 2
                },
                {
                  "sequenceNumber": 3
                }
              ]
            }
          }
        },
        {
          "sequenceNumber": 3,
          "epoch": {
            "epochId": 0,
            "checkpoints": {
              "nodes": [
                {
                  "sequenceNumber": 0
                },
                {
                  "sequenceNumber": 1
                },
                {
                  "sequenceNumber": 2
                },
                {
                  "sequenceNumber": 3
                }
              ]
            }
          }
        }
      ]
    }
  }
}
